### ISSUE TITLE
Track files other than Workflow.log in web interface

### ISSUE THREAD

**Issue by ashiklom** [PecanProject/pecan#1076]
It would be nice to have the option to follow other files, particularly those related to model execution (i.e. `stderr` and `stdout`.

**Comment by robkooper** [comment]
I agree, but the problem is I did not know how to do this cleanly with multiple runs as well as if it runs remotely (in which case the stdout/stderr are not transferred until the job is finished).

That being said, if you click on the finished button (even while the workflow is still running) you can see the stdout and stderr.
